guides/plugins/plugins/administration/using-utils.md

```diff
@@ -0,0 +1,59 @@
+# Using Utils
+
+Utils in the Shopware 6 Administration are registered to the Shopware object and are therefore accessible everywhere in the Administration.
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module. Of course you'll have to understand JavaScript. However, that's a prerequisite for Shopware as a whole and will not be taught as part of this documentation.
+
+## Accessing the Utils
+
+As noted in the introduction utils in the Shopware 6 Administration are bound to the Shopware global object. Learn more about that [here](./the-shopware-object.md).
+
+In this very short example we'll show you how to use a function of utils, in this case the `deepCopy` function.
```

Comment:
perhaps you should describe in a few words what `deepCopy` does

Reply:
I agree, please explain your example in more details, in your own words. Or at least use comments to emphasize the important aspects of this file. I don't like the usage of a plain file as example, without any explanation on it.

Reply:
well i changened the example and wrote a sentence about it